pyccel/codegen/printing/pycode.py

```diff
@@ -181,7 +178,7 @@ def _print_FunctionDef(self, expr):
         headers = expr.headers
         if headers:
             headers = self._print(headers)
-            code = '{header}\n{code}'.format(header=header, code=code)
+            code = '{header}\n{code}'.format(header=headers, code=code)
```

Comment:
Is this a bug-fix?

Reply:
Yes it was indicated by Codacy [here](https://app.codacy.com/gh/pyccel/pyccel/file/54639147259/issues/source?bid=15415567&fileBranchId=15415567#l184)

Reply:
Please add a comment to the PR's description then

Reply:
Done